stdlib/multiprocessing/context.pyi

```diff
@@ -27,11 +26,11 @@ class TimeoutError(ProcessError): ...
 class AuthenticationError(ProcessError): ...
 
 class BaseContext:
-    Process: type[BaseProcess]
-    ProcessError: type[Exception]
-    BufferTooShort: type[Exception]
-    TimeoutError: type[Exception]
-    AuthenticationError: type[Exception]
+    Process: ClassVar[type[BaseProcess]]
```

Comment:
I don't see this field, at least in Python 3.9.

Reply:
Hm, it exists here: https://github.com/python/typeshed/blob/df4618ef2dec1d27e363da0fc4ba8b00a2c811fe/stdlib/multiprocessing/context.pyi#L30

I just added a `ClassVar` here 🤔

Reply:
Doesn't exist in HEAD either: https://github.com/python/cpython/blob/75a6441718dcbc65d993c9544e67e25bef120e82/Lib/multiprocessing/context.py#L30

Let's just remove it.